### PR TITLE
Set search params in URL

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/pwl9HUcL/552-search-back-button-as-a-user-i-would-like-to-be-able-to-go-back-to-my-previous-search-using-the-browser-back-button-so-that-i-ca

### CODE REVIEW

**Review by paulmwatson — APPROVED** [review]

File: infrastructure/static/js/mm-webflow.js [hunk 1]
```diff
@@ -686,7 +697,29 @@ function mmWebflow(js) {
       }
```
Comment (paulmwatson) [inline infrastructure/static/js/mm-webflow.js:697]: Blank function?

File: infrastructure/static/js/mm-webflow.js [hunk 2]
```diff
@@ -686,7 +697,29 @@ function mmWebflow(js) {
...
     });
 
-    triggerSearch();
+    function onPopstate(event) {
+      loadSearchStateFromCurrentURL();
+      triggerSearch(listView.search.createUrl(), false);
+    }
+
+    function loadSearchStateFromCurrentURL() {
+      const queryString = window.location.search.substring(1);
```
Comment (paulmwatson) [inline infrastructure/static/js/mm-webflow.js:706]: Browsers have a good, reliable URL params method now and we don't need to chop it up ourselves. `URLSearchParams` https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

File: infrastructure/static/js/mm-webflow.js (hunk 1 above)
Comment (michaelglenister) [reply infrastructure/static/js/mm-webflow.js:697]: Looks fine to me, maybe Github minimized the code?

File: infrastructure/static/js/mm-webflow.js (hunk 2 above)
Comment (michaelglenister) [reply infrastructure/static/js/mm-webflow.js:706]: Changed. All I did was remove this nasty, redundant bit that manually chops up the queryString. The URLSearchParams approach is used inside `listView.loadSearchFromUrl(queryString);`

File: infrastructure/static/js/mm-webflow.js (hunk 1 above)
Comment (paulmwatson) [reply infrastructure/static/js/mm-webflow.js:697]: Oh dumb me, thanks.

**Review by paulmwatson — APPROVED** [review]

**Pull request merged**